src/orquestra/opt/optimizers/scikit_quant_optimizer.py

```diff
@@ -101,14 +101,15 @@ def _minimize(
             raise ValueError("TODO: reasonable message here")
 
         # if initial_params
-
-        result = scipy_minimize(
-            fun=cost_function,
-            x0=initial_params,
-            method=getattr(skq, self.method.lower()),
-            bounds=self.bounds,
-            options={"budget": self.budget},
-        )
+        with warnings.catch_warnings():
+            warnings.filterwarnings("ignore", category=DeprecationWarning)
```

Comment:
Might be best to only catch the depreciation warnings that we are already getting. So we know when new ones are cropping up.

Reply:
Its really hard to tell what are warnings we are already getting on the other basis than category. What if we remove <this> particular warning and the new one arises with the same category? Id have to somehow assert on the warning message which is not what I would like to see here